src/client.c

```diff
@@ -66,13 +88,23 @@ int send_request(int fd, char *hostname, char *port, char *path)
 {
   const int max_request_size = 16384;
   char request[max_request_size];
-  int rv;
 
-  ///////////////////
-  // IMPLEMENT ME! //
-  ///////////////////
+  int rv = recv(fd, request, max_request_size -1, 0);
 
-  return 0;
+  int resquet_length = sprintf(request,
+        "GET /%s HTTP/1.1\n"
+        "Host: %s:%s\n"
+        "Connection: close\n",
+        path, hostname, port);
+  
+  int rv = send(fd, request, request_length, 0);
+
+  if (rv < 0){
+    perror("send");
+    return;
```

Comment:
Since the `send_request` function has a return type of `int`, we can `return 0;` here